maven_iuvs/echelle.py

```diff
@@ -1892,10 +1908,10 @@ def writeout_l1c(light_l1a_path, dark_l1a_path, l1c_savepath, light_fits, binnin
     yMRH = 485 # the location of the row most-accurately representing the MRH altitudes across the aperture center (to be used by all emissions)
     yMRH = math.floor((yMRH-this_dict['ycH'].values[0])/this_dict['NbinsY'].values[0]) #  to get an integer value like IDL does.
 
-    H_brightnesses = [fit_params_list[i]["area_H"] for i in range(n_int)]
-    D_brightnesses = [fit_params_list[i]["area_D"] for i in range(n_int)]
-    H_1sig = [fit_unc_list[i]["unc_H"] for i in range(n_int)]
-    D_1sig = [fit_unc_list[i]["unc_D"] for i in range(n_int)]
+    H_brightnesses = [fit_params_list[i]['total_brightness_H'] for i in range(n_int)]
+    D_brightnesses = [fit_params_list[i]['total_brightness_D'] for i in range(n_int)]
+    H_1sig = [fit_unc_list[i]['unc_total_brightness_H'] for i in range(n_int)]
+    D_1sig = [fit_unc_list[i]['unc_total_brightness_D'] for i in range(n_int)]
```

Comment:
we should probably report the fitted IPH parameters in the l1c file, right?

Reply:
Oh nooooo, yes

Reply:
Fixed with c9545179fb52629ce74c4971ee40e51fe6117a2b

Reply:
Fixed in c9545179fb52629ce74c4971ee40e51fe6117a2b